eo-parser/src/main/resources/org/eolang/parser/cti/cti-adds-errors.xsl

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+The MIT License (MIT)
+
+Copyright (c) 2016-2023 Objectionary.com
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included
+in all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+-->
+<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform" id="cti-adds-errors" version="2.0">
+  <!--
+  For every cti objects add error messages.
+  -->
+  <xsl:output encoding="UTF-8" method="xml"/>
+  <xsl:template match="/program/errors">
+    <xsl:copy>
+      <xsl:apply-templates select="node()|@*"/>
+      <xsl:for-each select="//o[@base='cti']">
```

Comment:
@mximp Sorry I don't understand your concern here. Could you provide more examples, please?

Reply:
@mximp I got your idea. Thanks! I see several options here:
1. Yes, we expect that all libraries that we are using are clear and without CTI 
2. We can leave "errors" until runtime
3. We can check CTI only for application objects (exclude libraries).

Reply:
@mximp What do you think?

Reply:
@volodya-lombrozo Do you think it's possible to inspect only those objects which are actually used in current project? I believe that idea of CTI is wider than just marking deprecated code. We may want to mark (as warning) methods which might affect performance. It means that in general case the compiler must process only those CTI objects which are reachable from any object of current program.

Otherwise we can leave your implementation but set action to |"warning".

#3 from you list seems to make CTI useless.

Reply:
In general the following code does not make sense in case we react to any CTI:
```
  [] > foo
    cti > @
      2.times 2
      "error"
      "This method is deprecated!"
```

Since it will make the whole program/lib uncompilable.

Reply:
@mximp Ok, let's then handle all `cti` as warnings? We will just ignore `level` for now. In the future, when we will have enough information we will be able to create a new separate issue for that. Otherwise the current PR will be stuck.

What do you think?